content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -58,16 +58,16 @@ As mentioned earlier, Poseidon-Firmament scheduler enables an extremely high thr
 
 Although, Poseidon-Firmament scheduler is capable of scheduling various types of workloads, such as service, batch, etc., the following are a few use cases where it excels the most:
 
-1. For “Big Data/AI” jobs consisting of large number of tasks, throughput benefits are tremendous.  
-2. Service or batch jobs where workload resource requirements are uniform across jobs (Replicasets/Deployments/Jobs).  
+1. For “Big Data/AI” jobs consisting of large number of tasks, throughput benefits are tremendous.
+2. Service or batch jobs where workload resource requirements are uniform across jobs (Replicasets/Deployments/Jobs).
 
-## Current Project Stage	
+## Current Project Stage
 
-- We are **not aware of any production deployment** of Poseidon-Firmament scheduler at this time.  
-- Poseidon-Firmament is supported from Kubernetes release 1.6 and works with all subsequent releases.   
+- We are **not aware of any production deployment** of Poseidon-Firmament scheduler at this time.
+- Poseidon-Firmament is supported from Kubernetes release 1.6 and works with all subsequent releases.
```

Comment:
nit: Throughout the page, the scheduler is referred to as `Poseidon-Firmament` not `Poseidon/Firmament`. Are the names interchangeable?

Reply:
@deepak-vij @dhilipkumars Please check this.

Reply:
Yeah, that should be fine.

Reply:
Yes, names are interchangeable. Although, let us stick to Poseidon-Firmament naming convention for all the documentation going forward.

Reply:
Tidied